src/components/CreateCoursePage/index.jsx

```diff
@@ -12,6 +12,9 @@ class CreateCoursePage extends React.Component {
   constructor(props) {
     super(props);
     this.handleCourseCreate = this.handleCourseCreate.bind(this);
+    // We do not want to accidentally persist an error from a different page
+    // when a user comes to a create page.
+    if (props.courseInfo.error) props.clearCourseInfoErrors();
```

Comment:
If we're on the create page, don't we want to just clear everything, not just the error? We don't have any use for the data in courseInfo. It'll be overridden in happy path on successful create too, anyway.

Reply:
Uhmm... Sure, I'll update.

Reply:
If you do this, it could be like, `RESET_COURSE_INFO` or something like that - I think we have another reset action lurking around you could ape.

And you could simply do:
```
case RESET_COURSE_INFO:
    return initialState;
```

Reply:
So it looks like we can't clear out everything because the createCourseRunPage uses the courseInfo data.

Reply:
OK, well then forget everything I said (what does it use?)

Reply:
`const title = courseInfo.data && courseInfo.data.title ? courseInfo.data.title : '';
 const uuid = courseInfo.data && courseInfo.data.uuid ? courseInfo.data.uuid : '';`